src/components/date-picker/date-picker.scss

```diff
@@ -1,5 +1,7 @@
+@import '../../style/internal/variables';
+
 limel-portal {
```

Comment:
this whole block of styles with `limel-portal` is unnecessary now, since we have moved these styles to `portal.scss` line 1. This happened after the latest fixes of the portal. 

Also, there is no need for having a `z-index` here really I think. Probably `pointer-events: none;` would have been a better solution from the beginning to avoid blocking interactions (which is now a part of styles in `portal.scss`).

Reply:
⚡